apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -209,6 +209,23 @@ handle_request_('GetTokenSupplyByHeight', Req, _Context) ->
             {400, [], #{reason => <<"Chain too short">>}}
     end;
 
+handle_request_('DeleteTxFromMempool', Req, _Context) ->
+    ParseFuns = [ parse_map_to_atom_keys(),
+                  read_required_params([hash]),
+                  api_decode([{hash, hash, tx_hash}]),
+                  fun(_Req, #{hash := TxHash}) ->
+                      case aec_tx_pool:delete(TxHash) of
+                          ok ->
+                              {ok, {200, [], #{<<"status">> => <<"deleted">>}}};
+                          {error, already_accepted} ->
+                              {ok, {200, [], #{<<"status">> => <<"included">>}}};
```

Comment:
Feels a bit weird to have this as a "success" case. It's not in the mempool at this point so this feels more like another 404 case. Maybe with a different reason instead of not_found